app/Models/Device.php

```diff
@@ -396,6 +396,11 @@ public function getStatus()
         return $this->status;
     }
 
+    public function getLastPolled()
```

Comment:
last_polled should be a carbon object, so you don't need this function.

Reply:
I cannot access $device->last_polled from DeviceController.php: 
OK: ```'uptime' => Time::formatInterval($device->getStatus() ? $device->uptime : time() - $device->getLastPolled(), 'short'),```
NOK:``` 'uptime' => Time::formatInterval($device->getStatus() ? $device->uptime : time() - $device->last_polled, 'short'),```

Reply:
Try $device->lastPolled

Reply:
Still not. This one does not fail but the $device->lastPolled is empty.